src/OrbitGl/CMakeLists.txt

```diff
@@ -181,16 +181,20 @@ endif()
 add_executable(OrbitGlTests)
 
 target_sources(OrbitGlTests PRIVATE
+               CaptureViewElementTester.h
                PickingManagerTest.h
                UnitTestSlider.h)
 
 target_sources(OrbitGlTests PRIVATE
                BatcherTest.cpp
                CaptureStatsTest.cpp
+               CaptureViewElementTest.cpp
+               CaptureViewElementTester.cpp
                CaptureWindowTest.cpp
                GlUtilsTest.cpp
                GpuTrackTest.cpp
                MultivariateTimeSeriesTest.cpp
```

Comment:
Suppose you missed to add ThreadTrack and GpuTrack ones

Reply:
`GpuTrackTest.cpp` is already there, but I missed `ThreadTrackTest.cpp`, thanks!